HACKING.md

```diff
@@ -160,6 +160,36 @@ your charm to a controller using that version of Juju. For example, with microk8
 We rely on automation to [update charm pins](.github/actions/update-charm-pins/) of
 a bunch of charms that use the operator framework. The script can be run locally too.
 
+# Contributing
+
+Changes are proposed as [pull requests on GitHub](https://github.com/canonical/operator/pulls).
+
+Pull requests should have a short title that follows the
+[conventional commit style](https://www.conventionalcommits.org/en/) using one of these types:
+
+* chore
+* ci
+* docs
```

Comment:
🤔 I tend to open PRs with `docs:` prefix and Tony with `doc:` prefix.

Should we settle on a single type or allow both?

Reply:
Yes, let's go with `docs`, as that's what https://www.conventionalcommits.org/en/ recommends.

Reply:
Yeah, I've used `docs:` except I think just once, which was a typo on my part :disappointed:. This is where automation (in a follow-up PR, at some point in the future) would indeed be useful, to catch such errors.